homeworks/HW1/src/InputValidatedField.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {Input} from 'reactstrap';
+import './App.css';
+
+export class InputValidatedField extends React.Component {
+    constructor(props) {
+        super(props);
+        const error = '';
+        this.state = {
+            error: error
```

Comment:
Please check that repeat. Because I was written about this and you are resolved that discussion but never changed.